pkg/controller/job/job_controller.go

```diff
@@ -123,11 +121,13 @@ type Controller struct {
 	broadcaster record.EventBroadcaster
 	recorder    record.EventRecorder
 
-	podUpdateBatchPeriod time.Duration
+	syncJobBatchPeriod time.Duration
```

Comment:
we don't need this variable if the feature cannot be disabled. You can directly use the constant.

Reply:
Yes, however we currently have it zeroed in 8 unit tests, which would require adjusting: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/job/job_controller_test.go#L3392-L3393.
So, we could either (1) keep as is, (2.) use a package-level var updated by unit tests (which is what we do for backoff vars), (3.) adjust the unit tests to pass time. I prefer (3.), but I'm thinking if this should be a separate PR, because passing time in fake clock may introduce flakes.

Reply:
Yeah, 3 should be preferred. +1 in a separate PR